packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -339,3 +339,13 @@ export function char2Bytes(str: string) {
 export function bytes2Char(hex: string): string {
   return Buffer.from(hex2buf(hex)).toString('utf8');
 }
+
+/**
+ *
+ * @description Strips the first 2 characters of a hex string (0x)
+ *
+ * @param hex string to strip prefix from
+ */
+export function stripHexPrefix(hex: string): string {
+  return hex.slice(2);
```

Comment:
Really minor suggestion, the validation `.startsWith('0x')` could be part of the `stripHexPrefix` function and the original string can be returned if it doesn't start with 0x. It will avoid doing the validation `val.startsWith('0x')` every time the function is called.

Reply:
sounds good, I will rework that function